tests/_utils/run_br

```diff
@@ -20,4 +20,4 @@ br.test -test.coverprofile="$COV_DIR/cov.$TEST_NAME.$$.out.log" DEVEL "$@" \
     -L "info" \
     --ca "$TEST_DIR/certs/ca.pem" \
     --cert "$TEST_DIR/certs/br.pem" \
-    --key "$TEST_DIR/certs/br.key"
+    --key "$TEST_DIR/certs/br.key" \
```

Comment:
Maybe keep the empty line at the end of file.

Reply:
right, I added logic and later removed, need to revert it